src/tasks/nav.html

```diff
@@ -0,0 +1,3 @@
+<nav>
+    <a id="tasks.nav.settings" ui-sref-active="active" ui-sref="tasks.settings">{{'task.settings.title'|translate}}</a>
```

Comment:
@palbecki Let's use `task.tab.settings` variable here to have only 'Settings' instead of 'Tasks Settings'